### PR TITLE
Code Formatting for 1up Keyboards

### PR DESCRIPTION
**Pull request by Curry**
## Description

Using the references of noroadsleft's PR, #7820 , I have formatted the the default files to match that of noroadsleft.

* Removed all unused *_user & *_kb functions
* Formatted rules.mk to match that of the template.
* Formatted info.json files and separated by row.
* Formatted keymap.c files to look like the physical keymap.

User keymaps are untouched in this PR.

I am tagging the maintainers of each of these keyboards for their sign-off: 
1up60hse, super16: @mechmerlin
1up60hte: @bubnick 
1up60rgb: @rempired 
sweet16: @skullydazed 

## Types of Changes

- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).

### CODE REVIEW

**Comment by Curry** [comment]
@fauxpark I was debating doing that, but some of the user keymaps have it defined, and I figured to remove it from those keymaps, it would turn this into a breaking change.

Additionally, the build failure exists on master as well.

File: keyboards/1upkeyboards/1up60hse/info.json [hunk 1]
```diff
@@ -6,7 +6,72 @@
...
+                { "label": "T", "x": 5.5, "y": 1 },
+                { "label": "Y", "x": 6.5, "y": 1 },
+                { "label": "U", "x": 7.5, "y": 1 },
+                { "label": "I", "x": 8.5, "y": 1 },
+                { "label": "O", "x": 9.5, "y": 1 },
+                { "label": "P", "x": 10.5, "y": 1 },
+                { "label": "{", "x": 11.5, "y": 1 },
+                { "label": "}", "x": 12.5, "y": 1 },
+                { "label": "|", "x": 13.5, "y": 1, "w": 1.5 },
+                { "label": "Caps Lock", "x": 0, "y": 2, "w": 1.75 },
```
Comment (noroadsleft) [inline keyboards/1upkeyboards/1up60hse/info.json]:
```suggestion

                { "label": "Caps Lock", "x": 0, "y": 2, "w": 1.75 },
```

File: keyboards/1upkeyboards/1up60rgb/info.json [hunk 2]
```diff
@@ -1,27 +1,362 @@
...
+                { "label": "F", "x": 4.75, "y": 2 },
+                { "label": "G", "x": 5.75, "y": 2 },
+                { "label": "H", "x": 6.75, "y": 2 },
+                { "label": "J", "x": 7.75, "y": 2 },
+                { "label": "K", "x": 8.75, "y": 2 },
+                { "label": "L", "x": 9.75, "y": 2 },
+                { "label": ":", "x": 10.75, "y": 2 },
+                { "label": "@", "x": 11.75, "y": 2 },
+                { "label": "~", "x": 12.75, "y": 2 },
+                { "label": "Enter", "x": 13.75, "y": 2, "w": 1.25, "h": 2 },
```
Comment (noroadsleft) [inline keyboards/1upkeyboards/1up60rgb/info.json]:
```suggestion
                { "label": "Enter", "x": 13.75, "y": 1, "w": 1.25, "h": 2 },
```

File: keyboards/1upkeyboards/1up60rgb/info.json [hunk 3]
```diff
@@ -1,27 +1,362 @@
...
+                { "label": "C", "x": 4.25, "y": 3 },
+                { "label": "V", "x": 5.25, "y": 3 },
+                { "label": "B", "x": 6.25, "y": 3 },
+                { "label": "N", "x": 7.25, "y": 3 },
+                { "label": "M", "x": 8.25, "y": 3 },
+                { "label": "<", "x": 9.25, "y": 3 },
+                { "label": ">", "x": 10.25, "y": 3 },
+                { "label": "?", "x": 11.25, "y": 3 },
+                { "label": "Shift", "x": 12.25, "y": 3, "w": 2.75 },
+                { "label": "Ctrl", "x": 0, "y": 4, "w": 1.25 },
```
Comment (noroadsleft) [inline keyboards/1upkeyboards/1up60rgb/info.json]:
```suggestion

                { "label": "Ctrl", "x": 0, "y": 4, "w": 1.25 },
```

**Comment by mechmerlin** [comment]
I actually prefer the keymaps to not match the physical layout. I'm not in favor of those changes going in

**Pull request closed without merging**